src/narrow-screen/components/FilterSlideUpPanel.js

```diff
@@ -206,51 +206,47 @@ function SlideUpPanelBody(props) {
   )
 }
 
-export default class extends React.Component {
+export default (props) => {
 
-  state = {
-    selected: new Set(),
-  };
+  const [selected, setSelected] = useState(new Set());
 
-  onExpansionChanged() {
-    const expanded = this.props.expanded;
-    this.props.onExpansionChanged(!expanded);
+  function onExpansionChanged() {
```

Comment:
doesn't use => functions in 2020 **yikes** 

lol jk

Reply:
@cchyung LMAO bro don't you think that `function` work better with named functions? Anonymous ones are better off with `=>` imo - my preference lol